typescript/src/start_script.ts

```diff
@@ -104,14 +105,20 @@ try {
   }
   // copy proto file to generated folder
   const protoList = path.join(outputDir, 'proto.list');
+  const protoFilesMap = new Map<string, string>(
```

Comment:
I just realized you actually just need a `Set`, not a `Map`, and it can be initialized with a simple list.